testing/environments/elasticsearch/pki/generate_pki.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# Take the certificates and create a DER format and create a sha256 of it and encode it to base 64
+# https://www.openssl.org/docs/manmaster/man1/dgst.html
+openssl x509 -in ca/ca.crt -pubkey -noout | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | openssl enc -base64
```

Comment:
doesn't have to be for this PR, but how about we move to generating the keys & certs for the CA & elasticsearch instance?  It ends up being nice documentation for someone to see how to do it, and it will avoid vulnerability scanners complaining that we have keys checked into our repository.

Reply:
Hmm, I have a mild preference for static files since it makes it easy to point local configurations at the test container without having to worry about changing auth data, but if we can get it to play nice with that sort of testing it could be a good refactor

Reply:
lets revisit in the future.